hbase-client/src/main/java/com/yahoo/omid/transaction/HBaseAsyncPostCommitter.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2011-2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.omid.transaction;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import com.yahoo.omid.tsoclient.CellId;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class HBaseAsyncPostCommitter implements PostCommitActions {
+
+    private PostCommitActions syncPostCommitExecutor;
+
+    ExecutorService postCommitExecutor =
+            Executors.newSingleThreadExecutor(new ThreadFactoryBuilder().setNameFormat("postCommit-%d").build());
+
+    public HBaseAsyncPostCommitter(PostCommitActions postCommitExecutor) {
+        syncPostCommitExecutor = postCommitExecutor;
```

Comment:
I'll rename:
- postCommitExecutor to postCommitter
and
- syncPostCommitExecutor to syncPostCommitter

in order to avoid confusions with the real executor service from the Java concurrent library.